### PR TITLE
Remove some dead Makefile commands / scripts

### PR DESCRIPTION
**Pull request by cdrini**
Refactor: remove some dead code from the makefile

### Technical
- the solr commands are using an ancient version of solr
- all the other commands/scripts are unreferenceds
### Testing
Checked openlibrary and olsystem:
- ✅ No refs to `install_solr`
- ✅ No refs to `PYBUNDLE_URL`
- ✅ No refs to `SOLR_VERSION`
- ✅ No refs to `OL_VENDOR`
- ✅ No refs to `make.*venv`
- ✅ No refs to `setup_coverstore`
- ✅ No refs to `setup_ol`
- ✅ No refs to `make.*bootstrap`
- ✅ No refs to `make.*run`
- ✅ No refs to `make.*destroy`
- ✅ No refs to `commit_inside`
- ✅ No refs to `install_dependencies`

### CODE REVIEW

**Comment by cdrini** [comment]
Travis is being weird; going to try closing/reopening to rebuild.

File: Makefile [hunk 1]
```diff
@@ -48,45 +44,11 @@ distclean:
...
-setup_ol: git
-	@echo "** setting up openlibrary webapp **"
-	$(PYTHON) scripts/setup_dev_instance.py --setup-ol
-	@# When bootstrapping, PYTHON will not be env/bin/python as env dir won't be there when make is invoked.
-	@# Invoking make again to pick the right PYTHON.
-	make all
-
-bootstrap: venv install_solr setup_coverstore setup_ol
-
-run:
```
Comment (hornc) [inline Makefile]: Deleting this surprised me and seemed wrong, but it looks like we really don't use it anywhere, and rely on various scripts (e.g. https://github.com/internetarchive/openlibrary/blob/master/docker/ol-docker-start.sh )

It's be nice to have a single reliable way to start OL, and having it in the Makefile doesn't seem inappropriate. I guess we are not there yet, and sorting out how to run via docker in production is the real goal, + making dev representative of production.

This is just a comment, not really an objection to the change, I have checked and I don't think it'll cause problems (but was surprised!).

Comment (cdrini) [reply Makefile]: Yeah, took me by surprise as well; but this must've been before gunicorn; or maybe it was only ever intended for a development environment ¯\\_(ツ)\_/¯

File: Makefile [hunk 2]
```diff
@@ -48,45 +44,11 @@ distclean:
...
-
-install_solr:
-	@echo "** installing solr **"
-	mkdir -p var/lib/solr var/cache usr/local
-	wget -c $(OL_VENDOR)/$(SOLR_VERSION).tgz -O var/cache/$(SOLR_VERSION).tgz
-	cd usr/local && tar xzf ../../var/cache/$(SOLR_VERSION).tgz && ln -fs $(SOLR_VERSION) solr
-
-setup_coverstore:
-	@echo "** setting up coverstore **"
-	$(PYTHON) scripts/setup_dev_instance.py --setup-coverstore
```
Comment (mekarpeles) [inline Makefile]: This seems like a good command to document somewhere (e.g. wiki) as we haven't had to stand up a coverstore service for dev or production (and I imagine we soon may)
@cdrini

Comment (mekarpeles) [reply Makefile]: @anandology deprecated in 2015, we're safe to rm reference.
https://github.com/internetarchive/openlibrary/commit/6c00c6d7084602991cfcd3aa854b350e6ab80dd5#diff-4d3a94d17591ed9f3d442836fb4f9e4c

Thank you @cdrini for this archaeology

**Pull request merged**